build-package.sh

```diff
@@ -168,10 +168,26 @@ if [ "$(uname)" == "Darwin" ]; then
   fi
 fi
 
-# Generate jazzy docs (macOS)
-if [ "${docs}" == "true" ] && [ "$(uname)" == "Darwin" ]; then
-    sourceScript "${projectFolder}/Package-Builder/jazzy.sh"
+# Generate jazzy docs (macOS) where the 'documentation' tag exists on the issue label
+if [ "$(uname)" == "darwin"* ] && [ "${TRAVIS_PULL_REQUEST}" != "false" ]; then
```

Comment:
```
macbook:~ djones6$ uname
Darwin
```
I think this doesn't work, because you need an uppercase `D` for `Darwin`.

Reply:
I agree doesn't work, just tested it in a script on mac... I missed it when reviewing... Actually the if statement above shows a capital `Darwin`